test/config.js

```diff
@@ -1,91 +1,10 @@
 module.exports = {
-
-    TestFrontendUrl: process.env.TEST_URL || 'http://localhost:4000',
```

Comment:
This is required for the smoke tests. This should be put in the testing.yaml config file.